client/lib/domains/whois/test/assembler.js

```diff
@@ -6,105 +6,144 @@ const expect = require( 'chai' ).expect;
 /**
  * Internal dependencies
  */
-const whoisAssembler = require( './../assembler' );
+const whoisAssembler = require( './../assembler' ),
```

Comment:
This could be a good opportunity to switch to using the  import ____ from '_____' syntax :D